collaboration-and-sharing/README.md

```diff
@@ -48,6 +55,39 @@ An asset or media collection can be shared with users in the following ways:
 
    For example, if a user creates a folder, they can perform all operations that "Can manage" permission level allows on the folder and all the assets inside that folder. Even if you revoke the access through which they were able to create the folder in that directory, they would still have access with that permission level to that particular asset and any subfolders and files inside it.
 
+### Understanding how user's role change affects access
+
+When a user's role is changed from "Restricted media library access" to some other role or when any such user is deleted, they are removed from the user group. If their role is changed back to "Restricted media library access", they would still have access to all the assets and media collections that were previously individually shared with them. However, they won't be added back to any of the user groups they were previously added to.
+
+### Understanding how deleting user groups affects access
+
+When a user group is deleted, users lose access to any files, folders, or media collections that they had access to through the group.
+
+## Recommended workflow
+
+It is important to plan out your DAM workflow carefully. Your media collections, folders, and files should be organised such that your team members have appropriate access and permissions to the right resources.
+
+To help you establish your workflow, we recommend the following steps:
+
+1. Invite users and define your teams.
+   Firstly, invite users with "Restricted media library access" into your account. Consider the various teams in your organization. Do you, for example, have a design, marketing, sales, and technology team? This lets you think in terms of different teams and what sort of permission and access they will need in the next step. Once you have determined that, you can add them to their respective user groups.
+
+2. Organize your assets and media collections.
```

Comment:
Consider swapping steps 1 and 2. Create folder structure first, invite team members after.

Reply:
done